drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -407,11 +410,14 @@ public void checkRegisteredNodeMouseDoubleClickHandlerNotOverHeader() {
     public void link() {
         final GuidedDecisionTableView.Presenter dtPresenter2 = mock(GuidedDecisionTableView.Presenter.class);
         final GuidedDecisionTableView.Presenter dtPresenter3 = mock(GuidedDecisionTableView.Presenter.class);
-        final Set<GuidedDecisionTableView.Presenter> dtPresenters = new HashSet<GuidedDecisionTableView.Presenter>() {{
-            add(dtPresenter);
-            add(dtPresenter2);
-            add(dtPresenter3);
-        }};
+        final Set<GuidedDecisionTableView.Presenter> dtPresenters = new HashSet<GuidedDecisionTableView.Presenter>() {
```

Comment:
Is this Eclipse's formatting of static initialiser blocks (in anonymous sub-classes!)